docs/developer/install_local_customisations.md

```diff
@@ -0,0 +1,69 @@
+# Test your own Keptn build artefacts and/or custom helm chart configurations
+
+## Use Cases
+* As a developer I want to test my keptn fork build artefacts in a local kubernetes cluster environment.
+* As a developer I want to build and deploy my customized keptn fork at my kubernetes cluster environment.
+* As a keptn contributor I want to fork keptn, build, deploy and test my enhancements locally in my kubernetes cluster before I open a Pull Request.
+* As a mac book pro m1 (arm64) user, I want to build keptn locally and deploy it to my kubernetes cluster.
+* As a developer I want to set the repository and image tag at one point for all keptn artefacts.
+* ...
+
+## Customise Charts/Docker images and install with own values-local.yaml for local testing
+
+1. Prerequisites
+   - [Install Cluster](local_cluster_setup.md)
+2. Create `installer/manifests/keptn/values-local.yaml` file for your local values to be stored. The file should look like this:
+   - global keptn configuration: Set `global.keptn.registry` and `global.keptn.tag` if you did a local full build of keptn artefacts
+   - service configuration: If you only want to install one single artefact from your local build (e.g. apiService)
+```yaml
+# set global keptn registry and tag for completely overriding the keptn default config
+global:
+  keptn:
+    registry: "k3d-container-registry.127.0.0.1.nip.io:12345/keptn"      # keptn registry/image name
+    tag: "local-snapshot"                                                # keptn version/tag
+```
```

Comment:
shouldn't this part be under the global keptn configuration above? and then a second code block for the service specific config?